apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/SupplyDetailsGraph/SupplyDetailsGraph.tsx

```diff
@@ -9,13 +9,15 @@ import { StatisticsCard } from '../../../../2_molecules/StatisticsCard/Statistic
 import { useIsMobile } from '../../../../../hooks/useIsMobile';
 import { translations } from '../../../../../locales/i18n';
 import { Chart } from './components/Chart/Chart';
-import { harcodedData, LINE_COLOR } from './components/Chart/Chart.constants';
+import { harcodedData } from './components/Chart/Chart.constants';
 import { MockData } from './components/Chart/Chart.types';
 
 const pageTranslations = translations.aaveReserveOverviewPage.supplyDetails;
 
 type SupplyDetailsGraphProps = {};
 
+const LINE_COLOR = '#f58c31';
```

Comment:
As per figma seems that it's also a color from the pallete. So better use from there. This is `primary-30` and the green is `positive`. You can use the import as described other comment

Reply:
I aggree! I think we should  only hardcode those colors which are "border cases". 
We could also add some new colors to the palette if they appears many times as well.